apps/demos/Demos/Splitter/Overview/React/App.tsx

```diff
@@ -1,12 +1,21 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
 import Splitter, { Item } from 'devextreme-react/splitter';
 import PaneContent from './PaneContent.tsx';
 
-const PaneContentWithTitleRender = (title, initialSize?: string) => {
-  const PaneContentRender = (data) => (<PaneContent title={title} {...data} size={initialSize} />);
-  return PaneContentRender;
+const generate = (data,element,title, initialSize?: string) => {
+  const container = document.createElement('div');
+  ReactDOM.render(<PaneContent title={title} {...data} size={initialSize} />, container);
+  console.log(element);
```

Comment:
Don't forget remove this

Reply:
[da2d6de](https://github.com/DevExpress/DevExtreme/pull/27306/commits/da2d6de39f6fe219b3290ed0515b53976a732d19)